smartcontract/service/native/ontid/utils.go

```diff
@@ -68,6 +72,30 @@ func decodeID(data []byte) ([]byte, error) {
 	return data[1:], nil
 }
 
+func verifyID(id []byte) bool {
+	if string(id[0:8]) != "did:ont:" {
```

Comment:
obviously this will cause index out of bound panic .

Reply:
fixed